.github/workflows/README.md

```diff
@@ -38,4 +38,9 @@ Individual workflows with changes:
   - `ci_proposals_system_public.yml`: Runs the system specs for the public section
   - `ci_proposals_unit_tests.yml`: Runs the unit tests
 
-- `ci_performance_metrics_monitoring.yml`: runs Lighthouse metrics expectations against the app to get performance metrics
+- `ci_performance_metrics_monitoring.yml`: runs Lighthouse metrics expectations against the app to detect any performance regression. The expectations can be found in `lighthouse_budget.json`, where a time is defined for each metric:
```

Comment:
Maybe to keep things consistent, can we start it with a capital letter as the other ones in the same list?
```suggestion
- `ci_performance_metrics_monitoring.yml`: Runs Lighthouse metrics expectations against the app to detect any performance regression. The expectations can be found in `lighthouse_budget.json`, where a time is defined for each metric:
```